uvicorn/config.py

```diff
@@ -277,7 +277,7 @@ def __init__(
         self.interface = interface
         self.debug = debug
         self.reload = reload
-        self.reload_delay = reload_delay or 0.25
+        self.reload_delay = reload_delay
```

Comment:
There's no need to accept `None` for this. The behavior is not changed with this change.

Reply:
I agree it's cleaner but what if someone explicitly called it with `reload_delay=None`? The default value won't override it. Might be worth a mention in the changelog

Reply:
AFAIC, we shouldn't check a behavior that it's not allowed by the input, considering the type hint. I mean, `reload_delay` annotation changed from `Optional[float]` to `float` on this PR.

But... As you've pointed out, this is a breaking change, and I don't want to add breaking changes on this PR. 😅

I'll revert this change, and adapt the `run()`. 🙏

Reply:
Ok. I thought about it... Yep, you're right. I'll follow the path you suggested and add a note on the changelog about this.